src/app/app.functions/crm-card.js

```diff
@@ -1,23 +1,24 @@
 const axios = require("axios");
 
 exports.main = async (context = {}, sendResponse) => {
-  const fallbackQuote = `I think I do myself a disservice by comparing myself to Steve Jobs and Walt Disney and human beings that we've seen before. It should be more like Willy Wonka...and welcome to my chocolate factory.`;
```

Comment:
I don't think we need a fallback?

Reply:
agreed, we can throw the error message instead